lib/cmxl/fields/statement_details.rb

```diff
@@ -4,6 +4,12 @@ class StatementDetails < Field
       self.tag = 86
       self.parser = /(?<transaction_code>\w{3})(?<details>(?<seperator>.).*)/
 
+      class << self
+        def parse(line)
+          super line.gsub(/\n/, '')
+        end
+      end
```

Comment:
hmm. this is because we now split by `:` correct? but shouldn't we do that for all "lines"?

Reply:
for `:86:`, linebreaks are not a delimiter, so they can be ignored. that happend before implicitly via the `Cmxl.parse`-thingy, but has to be done explicitly now. 
other fields - besides `:61:` - should not have a line break at all within the fields body

Reply:
could add some dokumentation thought